CONTRIBUTING.md

```diff
@@ -126,6 +126,10 @@ room directly.
 
 You can also leave a message on our special [issue #1785](https://github.com/firefox-devtools/profiler/issues/1785).
 
+## Updating locales
+
+You can look at the [dedicated README file](./locales/README.md) in the locales directory.
```

Comment:
note: the rendered link is incorrect in the PR but I believe it will be right after merging.

Reply:
Are you rendering these outside of GitHub? Otherwise it looks good https://github.com/firefox-devtools/profiler/blob/74a5f6d26ca1c815eadfb20d46027a30c5eacf23/CONTRIBUTING.md

Reply:
ah yeah, this works when opening the file directly. This didn't work when using the button "display the rich diff" directly in the PR.